deploy_pulp3/pulp3/tasks/main.yml

```diff
@@ -0,0 +1,85 @@
+---
+# tasks file for pulp3
+- name: Create user {{ pulp_user }}
+  user:
+    name: "{{ pulp_user }}"
+    comment: "Pulp user"
+
+- name: Get user's home directory
+  shell: 'getent passwd "{{ pulp_user }}" | cut -d: -f6'
+  check_mode: false
+  changed_when: false
+  register: result
+
+- name: Set variable referencing user's home directory
+  set_fact:
+    pulp_user_home: "{{ result.stdout }}"
+
+- name: Make user's home directory world-readable
+  file:
+    path: "{{ pulp_user_home }}"
+    state: directory
+    mode: 0755
+
+- name: Create ~/.bashrc.d/
+  file:
+    path: "{{ pulp_user_home }}/.bashrc.d/"
+    state: directory
+  become_user: "{{ pulp_user }}"
+
+- name: Create directory /var/lib/pulp
+  file:
+    path: /var/lib/pulp
+    state: directory
+    owner: "{{ pulp_user }}"
+    group: "{{ pulp_user }}"
+    mode: 0775
+
+- name: Install pulpcore
+  pip:
+    name: pulpcore
+    state: latest
+    virtualenv: "{{ pulp_user_home }}/pulpvenv"
+    virtualenv_python: python3
+  become_user: "{{ pulp_user }}"
+
+- name: Add Django and Pulp supplemental bashrc
+  copy:
+    src: files/supplem.bashrc
+    dest: "{{pulp_user_home}}/.bashrc.d/suppelm.bashrc"
+
+- name: Source  ~/.bashrc.d/
+  blockinfile:
+    path: "{{ pulp_user_home }}/.bashrc"
+    block: |
+      if [ -d ~/.bashrc.d ]; then
+        for file in ~/.bashrc.d/*; do
+          . "$file"
+        done
+      fi
+    marker: "# {mark} Ansible managed block: source bashrc.d/"
+    create: yes
+  become_user: "{{ pulp_user }}"
+
+- name: Create /etc/pulp dir
+  file:
+    path: /etc/pulp
+    state: directory
+
+- name: Install server.yaml Config file
+  copy:
+    remote_src: true
+    src: "{{pulp_venv}}/lib/python3.5/site-packages/pulpcore/etc/pulp/server.yaml"
```

Comment:
Rather than hard-coding this path, one could use the `find` module to find `server.yaml`. This will help when supporting multiple versions of Python (say, 3.5 and 3.6).